src/main/resources/templates/IngrediëntenMagazijnScherm.html

```diff
@@ -25,19 +27,21 @@ <h3 id="modal-title" class="modal-title">Ingredient</h3>
             <div class="modal-body">
 
                 <form id="guestForm" enctype='text/plain'>
-
+                    <div ></div>
                     <div class="form-group">
-                        <label for="comment">Commentaar</label>
-                        <input id="comment" type="text" class="form-control" placeholder="Commentaar">
+                        <label for="ingredientName">Naam</label>
+                        <input id="ingredientName" type="text" class="form-control" placeholder="ingredientName">
                     </div>
 
                     <div class="form-check">
-                        <input class="form-check-input" type="checkbox" value="" id="ready">
-                        <label class="form-check-label" for="ready">
-                            is klaar
-                        </label>
+                        <label for="minimumStock">Minimum voorraad</label>
```

Comment:
Gebruik engelse benamingen, Minimum voorraad is niet engels. hetzelfde geldt voor huidige voorrad op regel 42